sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -614,3 +614,7 @@ CREATE TABLE `fee_schedule` (
     UNIQUE KEY `ins_plan_code_mod_type_date` (`insurance_company_id`, `plan`, `code`, `modifier`, `type`, `effective_date`)
 ) ENGINE=InnoDb AUTO_INCREMENT=1;
 #EndIf
+
+#IfMissingColumn form_encounter last_update
+ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
+#EndIf
```

Comment:
actually, this should not be needed since the upgrade mechanism with run patch.sql if it exists

Reply:
(so upgrade patch stuff should just need to go to database.sql and patch.sql)